test/utils/server.js

```diff
@@ -302,6 +302,14 @@ export default class TestServer {
 			res.destroy();
 		}
 
+		if (p === '/error/premature') {
+			res.writeHead(200, {'content-length': 50});
+			res.write('foo');
+			setTimeout(() => {
+				res.destroy();
+			}, 100);
+		}
+
```

Comment:
Is this somehow related to this PR or just a rebase problem?

Reply:
Yeah, this is not a part of the PR